lib/active_fedora/file_persistence.rb

```diff
@@ -0,0 +1,27 @@
+module ActiveFedora
+  module FilePersistence
+    extend ActiveSupport::Concern
+
+    include ActiveFedora::Persistence
+
+    private
+
+      def _create_record(_options = {})
+        return false if content.nil?
+        ldp_source.content = content
+        ldp_source.create do |req|
+          req.headers.merge!(ldp_headers)
+        end
+        refresh
+      end
+
+      def _update_record(_options = {})
+        return false unless content_changed?
```

Comment:
In 8fd3c606364b550b654b20fa227155ab3cc9ed7a, I've changed this from `true` to `false` (to cancel after_update callbacks, like we're doing above). It seems like this was introduced in c76c6ca37d58d386216f99c7c98a0acf0bf779d0, but it's unclear why create and update are different. @hectorcorrea @jcoyne?